content/en/news/releases/1.11.x/announcing-1.11/change-notes/index.md

```diff
@@ -0,0 +1,256 @@
+Services (MCS). This feature is off by default, but can be enabled by setting the
+`ENABLE_MCS_SERVICE_DISCOVERY` flag in Istio. When enabled, Istio will make service endpoints
+only discoverable from within the same cluster by default. To make the service endpoints within a cluster
+discoverable throughout the mesh, a `ServiceExport` CR must be created within the same cluster as the service
+endpoints. this process can be automated by enabling the Istio flag `ENABLE_MCS_AUTOEXPORT`. With this enabled,
+Istio will automatically create `ServiceExport` in all clusters for each service.
+  ([Issue #29384](https://github.com/istio/istio/issues/29384))
+
+- **Added** Specify `meshConfig.discoverySelectors` to dynamically restrict the set of namespaces for Services, Pods, and Endpoints that istiod processes when pushing xDS updates to improve performance on the data plane.
+  ([Issue #26679](https://github.com/istio/istio/issues/26679))
+
+
+- **Fixed** an issue to enableCoreDump using the sidecar annotation
+ ([reference]( https://istio.io/latest/docs/reference/config/annotations/)) ([Issue #26668](https://github.com/istio/istio/issues/26668))
+
+- **Fixed** an issue where app call itself by podIP in TPROXY mode could neither be intercepted by inbound nor outbound.
+  ([Issue #31095](https://github.com/istio/istio/issues/31095))
+
+- **Fixed** an issue where subject alternate names specified in service entry are not considered while building TLS context.
+  ([Issue #32539](https://github.com/istio/istio/issues/32539))
+
+- **Fixed** a bug where multiple gateways on the same port with `SIMPLE` and `PASSTHROUGH` modes is not working correctly.  ([Issue #33405](https://github.com/istio/istio/issues/33405))
+
+- **Fixed** a bug where Istio config gen fails when sum of endpoint weights over uint32 max.  ([Issue #33536](https://github.com/istio/istio/issues/33536))
+
+- **Fixed** smart DNS support in Istio CNI.
+  ([Issue #29511](https://github.com/istio/istio/issues/29511))
+
+- **Fixed** a bug in Kubernetes Ingress causing paths with prefixes of the form `/foo` to
+match the route `/foo/` but not the route `/foo`.
+  
+
+- **Fixed** an issue allowing ServiceEntries to act as instances in other namespaces.
+  
+
+- **Fixed** an issue causing proxies to send `Transfer-Encoding` headers with `1xx` and `204` responses.
+  
+
+- **Fixed** reconciliation logic in the validation webhook controller to rate-limit
+the retries in the loop. This should drastically reduce churn (and generated logs)
+in cases of misconfiguration.
+  ([Issue #32210](https://github.com/istio/istio/issues/32210))
+
+
+
+- **Optimized** generated routing configuration to merge virtual hosts with the same routing configuration. This improves performance for Virtual Services with multiple hostnames defined.
+  ([Issue #28659](https://github.com/istio/istio/issues/28659))
+
+
+## Security
+
+
```

Comment:
@istio/wg-security-maintainers anything to add to here?

Reply:
@myidpt @shankgan Anything to add here? Supporting multiple roots?

@yangminzhu @lizan Anything to add to the release notes?

Reply:
```
- **Added** validation for the `jwks` field in the request authentication policy. ([Issue #33053](https://github.com/istio/istio/issues/33053))
```